lib/gemstash/cli.rb

```diff
@@ -66,5 +67,20 @@ def status
     def stop
       Gemstash::CLI::Stop.new(self).run
     end
+
+    desc "preload", "Preloads all the gems in your gemstash server from the default upstream"
+    method_option :latest, :type => :boolean, :default => false, :desc =>
+      "Only fetch the latest specs"
+    method_option :server_url, :type => :string, :default => "http://localhost:9292", :desc =>
+      "Gemstash server url"
+    method_option :threads, :type => :numeric, :default => 20, :desc =>
+      "Number of threads to run the fetching job with"
+    method_option :limit, :type => :numeric, :default => nil, :desc =>
+      "Limit for the number of gems to preload"
+    method_option :skip, :type => :numeric, :default => 0, :desc =>
+      "Number of gems to skip when preloading"
+    def preload
```

Comment:
I wonder the same thing, on the one hand it makes sense to be shipped with gemstash as I like it to be a gemstash *feature* (another thing is how it manages to provide the feature), on the other hand it behaves as a batch client.

Maybe gemstash batch? and then we can use that batch for things like import gems in a batch and those other things that we talked about at some point?